### ISSUE TITLE
AWS::CloudWatch::Alarm.Period >= 60 for metrics in the AWS/ namespace

### ISSUE THREAD

**Issue by irvingpop** [aws-cloudformation/cfn-lint#1059]
*cfn-lint version: (`cfn-lint --version`)*
```
cfn-lint 0.22.3
```

*Description of issue.*

AWS won't let you set [`AWS::CloudWatch::Alarm.Period`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html#cfn-cloudwatch-alarms-period) less than `60`, it would be neat if cfn-lint flagged that.

Please provide as much information as possible:

The following resource should get flagged because the Period property is invalid, but doesn't:
```
  ELB5XXExceeded:
    Type: AWS::CloudWatch::Alarm
    Properties:
      AlarmName: !Sub AppELB_5XX_Exceeded-${AWS::StackName}
      AlarmDescription: Alarms when an 5xx requests exceed a specified threshold.
      AlarmActions: [!Ref AlertNotificationTopic]
      MetricName: HTTPCode_ELB_5XX_Count
      Namespace: AWS/ApplicationELB
      ComparisonOperator: GreaterThanOrEqualToThreshold
      EvaluationPeriods: 2
      Period: 30
      Statistic: Sum
      Threshold: 10
      Dimensions:
        - Name: LoadBalancer
          Value: !GetAtt ChefALB.LoadBalancerFullName
```

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

**Comment by PatMyron** [comment]
Good catch, reproduced:

`Period (30) not supported for namespaces with the following prefix: AWS/ (Service: AmazonCloudWatch; Status Code: 400; Error Code: ValidationError; Request ID: )`

```yaml
Resources:
  Alarm:
    Type: AWS::CloudWatch::Alarm
    Properties:
      ComparisonOperator: GreaterThanOrEqualToThreshold
      EvaluationPeriods: 2
      Period: 30
      MetricName: HTTPCode_ELB_5XX_Count
      Namespace: AWS/ApplicationELB
```

Wondering if that limitation is specific to the `AWS/` prefix from that error message
